### PR TITLE
try to clean disk space

### PR DESCRIPTION
**Pull request by jamesward**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -4,6 +4,10 @@ jobs:
   cloudshell_open:
     runs-on: ubuntu-latest
     steps:
+    - name: Free disk space
+      run: |
+        docker rmi $(docker image ls -aq)
```
Comment (ahmetb) [inline .github/workflows/ci.yml:9]: this can be done by doing `docker system prune -a -f`

Comment (ahmetb) [reply .github/workflows/ci.yml:9]: although it seems like this has resolved the issue :D

Comment (jamesward) [reply .github/workflows/ci.yml:9]:
```suggestion
        docker system prune -a -f
```

**Comment by ahmetb** [comment]
seems like it worked. https://github.com/GoogleCloudPlatform/cloud-run-button/pull/218/checks?check_run_id=2169463507

**Pull request merged**